### PR TITLE
normative statements: Conformance section

### PR DESCRIPTION
**Pull request by rhiaro**
Removes untestable normative language from Conformance section, towards #384

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/426.html" title="Last updated on Oct 18, 2020, 4:00 PM UTC (d3d5772)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/426/c764c67...d3d5772.html" title="Last updated on Oct 18, 2020, 4:00 PM UTC (d3d5772)">Diff</a>

### CODE REVIEW

File: index.html
Comment (brentzundel) [inline index.html:680]:
```suggestion
in Section <a href="#identifier"></a>which complies with relevant normative
```

File: index.html
Comment (brentzundel) [inline index.html:686]:
```suggestion
model described in this specification which complies with with the relevant
```

**Review by brentzundel — APPROVED** [review]
minor nits, otherwise LTGM

**Review by msporny — CHANGES_REQUESTED** [review]
Approved as long as the changes that @brentzundel requested are made.

**Review by peacekeeper — APPROVED** [review]
+1 to this with @brentzundel 's suggestions.

**Comment by msporny** [comment]
@rhiaro waiting on you to merge @brentzundel's changes to make sure you're ok with them.

**Review by kdenhartog — APPROVED** [review]
Looks like @brentzundel suggestions have been merged now as well

File: index.html
Comment (msporny) [reply index.html]: Merged.

File: index.html
Comment (msporny) [reply index.html]: Merged.

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Normative, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**
